app/charts/shared/chart-helpers.tsx

```diff
@@ -160,4 +168,43 @@ export const getLabelWithUnit = (
     : dimension.label;
 };
 
+export const checkForMissingValuesInSegments = (
+  groupedMap: InternMap<string, Array<Observation>>,
+  segments: Array<string>
+): boolean => {
+  for (const value of groupedMap.values()) {
+    if (value.length !== segments.length) {
+      return true;
+    }
+  }
+
+  return false;
+};
 
+export const useImputationNeeded = ({
```

Comment:
Do I understand correctly … this is only true IF we use segments? No imputation needed when the area chart is not stacked?

Reply:
Yes, right now this only applies to stacked area charts – in case of a regular area chart, there won't be any missing segments which would require imputation